lib/swagger/docs/generator.rb

```diff
@@ -91,6 +91,12 @@ def generate_doc(api_version, settings, config)
 
         private
 
+        def check_for_http_path(debased_path, api_version)
+          path = "#{Config.transform_path(trim_leading_slash(debased_path), api_version)}"
+          return path if path.include?("http")
```

Comment:
I think this should be more specific with a regex.
e.g.
`
return path if path.match(/^https?:\/\//)
`